modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/impl/WFSDataAccessFactory.java

```diff
@@ -576,6 +576,22 @@ public T lookUp(final Map params) throws IOException {
                                 name, String.class, title, description, null, "program");
     }
 
+    @SuppressWarnings("rawtypes")
+    public static final WFSFactoryParam<Map> ADDTIONAL_HEADERS;
+
+    static {
+        String name = "WFSDataStoreFactory:ADDTIONAL_HEADERS";
+        String title = "Set additional HTTP request headers.";
+        String description =
+                "The given HTTP headers will be set on HTTP requests in addition to "
+                        + "the regular GeoTools managed headers (like Accept-Enconding, Content-type, "
+                        + "User-Agent, Authorization). Provided values must not conflict with GeoTools "
+                        + "managed headers. Parameters must use String keys and values.";
+        parametersInfo[23] =
+                ADDTIONAL_HEADERS =
```

Comment:
Typo here

Reply:
Ups, thank you for finding!

Reply:
Fixed now. Dont know if I am expected to resolve this now...